### DISCUSSION TITLE
Add Next.js public folder to starters?

### DISCUSSION THREAD

**Discussion by gr-qft** [vercel/turborepo#225] [Help] [answered]
Thanks for this awesome library @jaredpalmer ! Great work! 

I have a question. I tested out with `npx create-turbo@latest`. I noticed that inside web or docs there's no `public` like a typical Next.js project. Is it a good thing to add a public folder inside (for images etc.), and if not, what should I do? 

Also, should I just modify tsconfig.json as usual for absolute imports etc. ? 

Thanks again!

**Comment by jaredpalmer** [comment, marked as answer]
Thanks for bringing this up.

I think People can add that on their own as well as paths. Right now, we want to remove as many files as we can to keep things simple from the start

**Reply by gr-qft** [reply]
Awesome, thanks a lot again for the info and for this great library!
